src/gsy_e/models/strategy/load_hours.py

```diff
@@ -114,10 +119,9 @@ def state(self) -> LoadState:
         return self._state
 
     def _init_price_update(self, fit_to_limit, energy_rate_increase_per_update, update_interval,
-                           use_market_maker_rate, initial_buying_rate, final_buying_rate):
+                           initial_buying_rate, final_buying_rate):
 
         # Instantiate instance variables that should not be shared with child classes
-        self.use_market_maker_rate = use_market_maker_rate
         self.fit_to_limit = fit_to_limit
```

Comment:
After looking at the code for the comment in the line right above, I found out that this member can be completely deleted. Feel free to do so, thanks!